docs/make.jl

```diff
@@ -124,13 +124,22 @@ end
 
 push!(allmods,Plots)
 
+mathengine = MathJax3(Dict(
```

Comment:
I've added this to Catalyst in the tutorial PR there to get Latex system printing working again in Documenter.